### PR TITLE
Config Deploy file to use it with multi environment

### PR DESCRIPTION
**Pull request by trung-ho**
Edit After Steven Feedback at 8 Oct

### CODE REVIEW

File: drills/deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+## Purpose of Deploy.rb
+This document for who using Mina gem at https://github.com/mina-deploy/mina
+
+This document describle how to use 1 deploy.rb file for multible environment.
```
Comment (stevenyap) [inline drills/deploy/README.md]: Spelling mistakes. Please rectify.

File: drills/deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+After installation is finished, run
+
+    $ mina init
+    Created config/deploy.rb.
+
+##Initialize Amazon web services for deploy.
+Contact adminstrator to setting up deploy environment on AWS.
+
+#Setting mina for environment
```
Comment (stevenyap) [inline drills/deploy/README.md]: This should be a H2, not a H1

File: drills/deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    
+    $ mina setup to=your_app_environment
+
+E.g: mina setting for staging environment
+    
+    $ mina setup to=staging
+
+##Setting common config for both Production environment and Staging environment.
+Open config/deploy.rb
```
Comment (stevenyap) [inline drills/deploy/README.md]: Please add more description to this step.
Why do you need to do this?

File: drills/deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    ruby_gemset       = 'your_app_gemset'
+
+    set :term_mode, nil
+    set :shared_paths, ['config/database.yml', 'log']
+    set :app, app
+    set :repository, "ssh://#{user}@#{domain}/~/git/#{app}.git"
+    set :user, user
+    set :rvm_path, "/usr/local/rvm/bin/rvm"
+
+##Setting individual config for each environment
```
Comment (stevenyap) [inline drills/deploy/README.md]: Please add more description to this step.
Why do you need to do this?

File: drills/deploy/README.md (hunk 2 above)
Comment (stevenyap) [inline drills/deploy/README.md]: Setup for mina should be done AFTER the deploy.rb is edited.

File: drills/deploy/README.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+    stage = ENV['to'] || 'staging'
+    case stage
+      when 'staging'
+        set :branch, 'master'
+        set :domain, staging_domain
+      when 'production'
+        set :branch, 'master'
+        set :domain, production_domain
+      else
+        print_error "Please specify a stage. eg. mina deploy to=production"
```
Comment (stevenyap) [inline drills/deploy/README.md]: I have spoke about this in the last revision.
Please rectify this.

File: drills/deploy/README.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+    set :rails_env, stage
+    set :deploy_to, "/home/#{user}/#{stage}"
+
+    invoke :"rvm:use[#{ruby_version}@#{ruby_gemset}]"
+
+##Push code to server
+After finish config, push your code to staging
+
+    $ git remote add your_environment ssh://ec2-user@[your_app_domain]/~/git/[your_app_name].git
```
Comment (stevenyap) [inline drills/deploy/README.md]: your_environment should be [your_environment]

File: drills/deploy/README.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+          exit
+      end
+
+      set :rails_env, stage
+      set :deploy_to, "/home/#{user}/#{stage}"
+
+      invoke :"rvm:use[#{ruby_version}@#{ruby_gemset}]"
+
+    end
+
```
Comment (stevenyap) [inline drills/deploy/README.md]: Unneeded lines

**Comment by stevenyap** [comment]
@chacharoti I have revised it. Please update it again.

Please don't close the PR. Just add the new changes in a new commit.
There are some previous comments I made and it is gone but the mistakes are still there.

Furthermore, please look through your whole page and write it better if possible.

**Comment by trung-ho** [comment]
@stevenyap  Hi Steven, I had updated as your request, please check.

File: drills/deploy/README.md (hunk 5 above)
Comment (stevenyap) [inline drills/deploy/README.md]: This is still not rectified @chacharoti

**Comment by stevenyap** [comment]
@jamesfwz Please help me to do a final review on this PR. This is Chacharoti's PR for deploying to Mina. There are a lot of better ways to express the English in this PR so I'll see what you can do. :)

**Comment by stevenyap** [comment]
@chacharoti please move the file to /protocols/drills/aws-deployment/README.md

**Comment by trung-ho** [comment]
@stevenyap I had changed directory of file

**Comment by stevenyap** [comment]
@chacharoti thanks. This PR is ready to merge. Please squash it into a single commit and I will merge it.

**Comment by trung-ho** [comment]
@stevenyap I had squared all commit to one commit. Please check

**Pull request merged**